includes/Core/Admin/Notice.php

```diff
@@ -125,7 +125,7 @@ public function render() {
 
 		?>
 		<div id="<?php echo esc_attr( 'googlesitekit-notice-' . $this->slug ); ?>" class="<?php echo esc_attr( $class ); ?>">
-			<?php echo $content; /* phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped */ ?>
+			<?php echo wp_kses_post( $content ); /* phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped */ ?>
```

Comment:
Do we still need the ignore comment here?

Reply:
Good catch, thanks. Removed it.